src/components/Contacts/ContactListTable.jsx

```diff
@@ -1,36 +1,32 @@
 // React Imports
-import React, { useContext } from 'react';
+import React from 'react';
 // Material UI Imports
 import Table from '@mui/material/Table';
 import TableBody from '@mui/material/TableBody';
 import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
-// Context Imports
-import { UserListContext } from '@contexts';
 // Component Imports
-import ClientListTableRow from './ClientListTableRow';
+import ContactListTableRow from './ContactListTableRow';
 import { StyledTableCell } from '../Table/TableStyles';
 
 // ===== MAKE CHANGES HERE FOR TABLE HEADER / COLUMN TITLES =====
-const columnTitlesArray = ['Client', 'Pin', 'Delete'];
+const columnTitlesArray = ['Contact', 'Pin', 'Delete'];
 
 /**
- * @typedef {import("../../typedefs.js").clientListTableProps} clientListTableProps
+ * @typedef {import("../../typedefs.js").contactListTableProps} contactListTableProps
```

Comment:
The typedef in `typedefs.js` would need to be changed as well along with the new props for `ContactListTable`

Reply:
Is it possible to move these into a file closer to the code, or enable automated validation of these? Devs will not be able to keep track of these types.

Reply:
Either way, I've updated things for this PR.

Reply:
moved the typedef into the ContactListTable and updated it.

Reply:
Would still like to see a followup to enable automated validation of JSDoc. I don't see much value in using JSDoc if we don't have some automated requirements for adding it.

Reply:
> Is it possible to move these into a file closer to the code, or enable automated validation of these? Devs will not be able to keep track of these types.

Yes, we could move the typedefs just above the type we're defining instead of importing it from one location.

As for keeping track of these types, I believe there's an eslint plugin we could use for JSDoc called `eslint-plugin-jsdoc`.

Reply:
There's a way to make it do typescript like validation (see screenshot below from eslint-plugin-jsdoc documentation):

![Screen Shot 2023-08-11 at 12 28 10 (2)](https://github.com/codeforpdx/PASS/assets/14917816/5f6d5934-16f3-4e4e-a1d2-56f54ae85c81)

Reply:
Cool. We can look into implementing this in a separate issue.